guide/contribute/case-studies.md

```diff
@@ -0,0 +1,114 @@
+
+### Visuals
+* A graphic showing the user persona the application is targeting
+* A list of questions the team used to come up with the user persona
+
+Example of user persona Blixt wallet case study
+{% include picture.html
+   image = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixt.png"
+   retina = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtX2.png"
+   mobile = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtMobile.png"
+   mobileRetina = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtMobileX2.png"
+   alt-text = "User personas example"
+   width = 1600
+   height = 800
+   modalWidthMobile = 400
+   modalHeightMobile = 400
+%}
+
+Example of user persona Payjoin case study
+
+
+## Methodology
+What steps led to the final solution? While the specific steps don't need to be detailed extensively, it's important to highlight the ones that provided the most valuable insights. Feel free to mention any challenges faced, key decision points regarding the chosen steps or tools.
+
+### Visuals
+* Images of userflows or snapshots of research done. Anything visual can go here that helps to tell a bit of a story. These can be messy.
+* An image showing the main insights, if you have one.
+
+## Insights
+After completing all the steps to solve the particular design challenge, you likely discovered some key insights. Share them here.
+
+### Content ideas
+* A blurb summarising what was learnt.
+* Bullet points highlighting the main insights.
+
+## Conclusion
+This is a final wrap-up where we close off the story. Share what happened after the case study/research was completed.
+
+### Content ideas
+* What was the impact of the solution? Add in qualitative and quantitative data if you have it.
+* Was the solution implemented by the project?
+* What were the next steps after completing this process?
+* Make sure to thank everyone who participated or helped
+
+## Resources
+This is the last section of the case study where it’s nice to share links to all the parts; or rather the puzzle pieces. Here you can add in simple titles such as a Figma file or links to research conducted. Add all the artefacts that were used in the entire design process to give the reader the option to dive in and keep learning from your experience.
+
+## Acknowledgements/Gratitude
+It is a good idea to acknowledge people, projects or organisations that provided resources, input, feedback etc. for the case study.
+
+Related resources:
+The below 2 case studies were put together roughly using the above guidelines.
+* [Blixt](/guide/case-studies/blixt-wallet/)
+* [The Payjoin experience](/guide/case-studies/payjoin/)
+
+---
+
+{% include next-previous.html
+   previousUrl = "/guide/review/"
+   previousName = "Review"
+   nextUrl = "/guide/daily-spending-wallet/"
```

Comment:
Next page is not right. Should be the [Glossary](https://bitcoin.design/guide/glossary/).